dev-tools/cloud/launch_cloud_e2e_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
It looks like this file is added entirely as it was added as part of https://github.com/elastic/fleet-server/pull/3243, We may need to add that backport here.

What do you think @mrodm @nchaulet?

Reply:
Not super familiar with that file but if we should probably try to backport it